.env.sample

```diff
@@ -1,3 +1,7 @@
+# Django storage
+DJANGO_STORAGE_DIR=.
+DJANGO_DB_FILE=django.db
```

Comment:
Let's add `*.db` to the `.gitignore` and `.dockerignore`

Reply:
Yep! It's in `.gitignore` and I added it to `.dockerignore` in b049f6a73870384e59c9af538e94e489098806b1